CMakeLists.txt

```
@@ -284,6 +284,7 @@ option(FLB_FILTER_NIGHTFALL            "Enable Nightfall filter"
 option(FLB_FILTER_WASM                 "Enable WASM filter"                           Yes)
 option(FLB_PROCESSOR_LABELS            "Enable metrics label manipulation processor"  Yes)
 option(FLB_PROCESSOR_ATTRIBUTES        "Enable atributes manipulation processor"      Yes)
+option(FLB_PROCESSOR_SELECTOR          "Enable selector processor"                    Yes)
```

I would invert the description here:
```suggestion
option(FLB_PROCESSOR_SELECTOR          "Enable processor extended selector"                    Yes)
```

---

I would like to use ambiguous description because we're planning to extend a capability to handle traces in the future.
Or, how about using "Enable selector for non-logs type of events" or something similar?

---

This processor is a non-log types parity of logs type handling of filter_grep.

---

Why not just make it a processor_grep then? That makes more sense to me, it feels like we're just working around the lack of grep support for metrics and traces.

So this won't work for logs then, so it's not really a processor itself but a specific type of metrics (only?) grep processor?

---

This is because normal processor_grep will be collided for name collisions. Because normal filters are able to use as processors.